tools/single_use/22.0_To_23.0.php

```diff
@@ -0,0 +1,133 @@
+<?php declare(strict_types=1);
+/**
+ * This script is used to correctly establish a link betwen the `issues` table
+ * used by the Issue Tracker module and the new table `modules`.
+ * It outputs SQL statements necessary to upgrade this table from LORIS version
+ * 22 to version 23.
+ * It relies on the existence of the `LorisMenu` table and the `modules` table
+ * so it must be used in the following way:
+ *
+ * 1. A user must apply the patch file:
```

Comment:
@ridz1208 @driusan Please let me know if this is the upgrade process you had in mind. I'm happy to modify the script to make the upgrade process smooth.

Reply:
We'll need to change the patch name before the release and make sure it's not duplicated in the release patch, but yes, this seems reasonable.

Reply:
adressed in my PR